README.md

```diff
@@ -0,0 +1,12 @@
+# STW Mapping Library
```

Comment:
can we please expand only at this place to give a view for about the project? thanks.